bin/setup

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env ruby
-require 'pathname'
-require 'fileutils'
+
+require "pathname"
 
 gem_root = Pathname.new(__dir__).join("..")
-spec_manageiq = gem_root.join("spec/manageiq")
```

Comment:
Rubocop warned about `spec_manageiq` being an unused variable here, but think we need to update this to the current `bin/setup` used by the other plugins